app/src/main/java/org/fossasia/openevent/general/model/Event.kt

```diff
@@ -10,52 +10,52 @@ import com.github.jasminb.jsonapi.annotations.Type
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
 data class Event(
         @Id(LongIdHandler::class)
-        var id: Long? = null,
-        var paymentCountry: String? = null,
-        var paypalEmail: String? = null,
-        var thumbnailImageUrl: String? = null,
-        var schedulePublishedOn: String? = null,
-        var paymentCurrency: String? = null,
-        var organizerDescription: String? = null,
-        var originalImageUrl: String? = null,
-        var onsiteDetails: String? = null,
-        var organizerName: String? = null,
-        var largeImageUrl: String? = null,
-        var timezone: String? = null,
-        var deletedAt: String? = null,
-        var ticketUrl: String? = null,
-        var locationName: String? = null,
-        var privacy: String? = null,
-        var codeOfConduct: String? = null,
-        var state: String? = null,
+        val id: Long? = null,
```

Comment:
Many fields can be non nullable

See https://github.com/iamareebjamal/openevent-java/blob/master/core/src/main/java/api/openevent/event/Event.kt for example